recOrder/compute/qlipp_compute.py

```diff
@@ -9,8 +9,8 @@ def initialize_reconstructor(
     wavelength_nm=None,
     swing=None,
     calibration_scheme=None,
-    NA_obj=None,
-    NA_illu=None,
+    NA_obj=1.0,
+    NA_illu=1.0,
```

Comment:
Why are these values set to 1.0/1.0 while the defaults in the reconstruction tab are set to 1.3/0.5?

Reply:
Good point, updated.